client/cli/cli.go

```diff
@@ -18,6 +20,7 @@ import (
 	"github.com/peterh/liner"
 )
 
+const configDir = ".dlv"
```

Comment:
Actually I see you're using os/user to get homedir which I think will give USERPROFILE.  https://golang.org/src/os/user/lookup_windows.go#L115